client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java

```diff
@@ -116,9 +119,11 @@ public static int getBombVariant(BombEntity bombEntity) {
         ItemType type = bombEntity.getBombItemType();
         switch (type) {
             case SMALL_BOMB:
-                return BombSprite.VARIANT_SMALL_RED; // TODO: Check the player texture variant
+                return BombSprite.VARIANT_SMALL_RED;
```

Comment:
Why was this TODO removed? The feature is still missing.

Reply:
This feature was deemed as an inefficient use of company resources at the 13.06.2020 board meeting, thus making this TODO unnecessary.

Reply:
Well in that case, this method is useless, as is the Bomb's playerTextureIndex property and variant constants, all of which should be removed.

Reply:
Removed